packages/forms-web-app/src/middleware/navigation-history.js

```diff
@@ -2,6 +2,7 @@ const config = require('../config');
 
 const defaultOptions = {
   fallbackPath: config.appeals.startingPoint,
+  filterPaths: [/favicon.ico/, /^\/document\//],
```

Comment:
ensure unwanted paths do not get thrown into the 'back' link functionality.

This is a bodge really, as the back link functionality itself is a bodge (imo). 

Perhaps a better system here would be a white list?